app/models/document.py

```diff
@@ -94,10 +100,18 @@ def status(self):
 
     @property
     def date_created(self):
```

Comment:
I don't think there will be multiple published or created events. However, I checked in with Brian and we decided that it would be easier for him to query if we have these properties as columns instead.